pennylane/templates/state_preparations/mottonen.py

```diff
@@ -235,6 +231,11 @@ def MottonenStatePreparation(state_vector, wires):
 
     This code is adapted from code written by Carsten Blank for PennyLane-Qiskit.
 
+    .. note::
+
+        Due to numerical errors stemming from finite precision, the final prepared state
+        is only an approximation of the input state vector. However, the fidelity with the desired state is
+
```

Comment:
Sentence ends abruptly

Reply:
Perhaps also note something about the global phase?

Reply:
Oh shoot, this is what I wanted to do and got distracted...